plugins/inputs/snmp/CONFIG-EXAMPLES.md

```diff
@@ -63,3 +63,69 @@ Here is the configuration that you add to your `telegraf.conf`:
       oid = "IF-MIB::ifDescr"
       is_tag = true
 ```
+
+
+See below for SNMPv3 Examples of configs against Cisco Switches: 
+
+```
+[[inputs.snmp]]
+  agents = [AGENTS]
+  version = 3
+  auth_password = "TOKEN"
+  auth_protocol = "MD5"
+  priv_password = "TOKEN"
+  priv_protocol = "AES"
+  sec_name = "test"
+  sec_level = "authPriv"
+  max_repetitions = 10
+  timeout = "10s"
+  retries = 1
+  name_prefix = "dev."
+  
+  [[inputs.snmp.field]]
+    name = "hostname"
+    oid = ".1.3.6.1.2.1.1.5.0"
+    is_tag = true
+  [inputs.snmp.tagdrop]
+    ifName=["Vlan[0-9]*", "loopback*"]
+    fieldpass = ["ifInUcastPkts", "ifInNUcastPkts", "ifInDiscards", "ifInErrors", "ifOutUcastPkts", "ifOutNUcastPkts", "ifOutDiscards", "ifOutErrors", "ifType", "ifMtu", "ifSpeed", "ifAdminStatus", "ifOperStatus"]
```

Comment:
A key "belongs" to the previously defined table, which means this falls under the tagdrop table.  For it to work as expected it needs to belong to the plugin table `[[inputs.snmp]]`.

Reply:
Not totally sure what you mean?  I realize that there was an extra [inputs.snmp.tagdrop] on line 95-96 and removed this.  Not sure why it was not producing an error, but could be because the value on line 96 was empty?

Also when you say "A key 'belongs`' to the previously defined table" -- what key are you referring to exactly?  There is an `[[inputs.snmp.table]]` on line 92 named "interfaceStats" -- so are you saying I need to move up the tag drop?   How do I go about putting the tag drop in the [[inputs.snmp]] table?  Is it just a matter of moving it up to above the first instance of the `[[inputs.snmp.field]]` entry on line 85?

I am a little confused on what key you are referring to and which table it should be put under?  Can you please clarify?

Reply:
In this example the `fieldpass` key is under the `tagdrop` table, in order for it to work though it needs to be part of the plugins table: `[[inputs.snmp]]`.  Indention has no meaning, its just for readability, so yes it needs to be above any subtables.

The config file is using [TOML](https://github.com/toml-lang/toml), so I suggest reading through this document for a better understanding.